modules/engines/engine-odata/src/test/java/org/eclipse/dirigible/engine/odata2/transformers/OData2ODataMTransformerTest.java

```diff
@@ -52,12 +53,14 @@ public void testTransform() throws IOException, SQLException {
         PersistenceTableColumnModel column1 = new PersistenceTableColumnModel("Id", "Edm.Int32", true);
         PersistenceTableColumnModel column2 = new PersistenceTableColumnModel("Customer", "Edm.String", false);
         PersistenceTableModel model = new PersistenceTableModel("ORDERS", Arrays.asList(column1, column2), new ArrayList<>());
+        model.setTableType(ISqlKeywords.METADATA_TABLE);
```

Comment:
Can we set in `PersistenceTableModel` by default `private String tableType = ISqlKeywords.METADATA_TABLE`?